x-pack/heartbeat/heartbeat.reference.yml

```diff
@@ -287,6 +287,10 @@ heartbeat.jobs:
 # sub-dictionary. Default is false.
 #fields_under_root: false
 
+# Configure the precision of all timestamps in Heartbeat.
```

Comment:
Am I right in understanding that this basically just changes how `time.Time` fields are serialized when used with `time.Time`?  If so, this should work.

However, we should add a note that time fields will keep their `us` (microsecond) paths, such as `monitor.duration.us`.

Reply:
Nevermind, I think we should be fine here, all the `us` stuff is durations not timestamps.